# Delete an existing group

string group_name
----
string feedback